### DISCUSSION TITLE
[Feature Request] Does this work with other discord bots or just GiveawayBot?

### DISCUSSION THREAD

**Discussion by Sudowo** [marioparaschiv/nitro-sniper#65] [Ideas]
I tried to test this with Santa Lunar (one that I usually use) and it does not work. Does this only work with GiveawayBot? If so, can you add support for most of the other discord bots that are used for giveaways? (Invite Tracker, Prism, Plasma, etc,)

**Comment by ghost** [comment]
Bot Tag: tip.cc#7731
Bot ID: 617037497574359050
Invite Link: https://discordapp.com/oauth2/authorize?client_id=617037497574359050&scope=bot%20applications.commands&permissions=344128
Default Prefix & Giveaway Start Command: $airdrop

**Comment by finaviggo** [comment]
Bot Tag: Giveaways#8717
Bot ID: 651627261782523926
Invite Link: https://discord.com/oauth2/authorize?client_id=651627261782523926&permissions=379968&scope=bot
Default Prefix & Giveaway Start Command: %start <duration> [winners] <prize>

**Comment by klendite** [comment]
Bot Tag: Plasma#3518
Bot ID: 716967712844414996
Invite Link: https://discord.com/oauth2/authorize?client_id=716967712844414996&permissions=8&scope=bot
Default Prefix & Giveaway Start Command: .gstart

**Comment by klendite** [comment]
Bot Tag: Noumenon#7110
Bot ID: 634866217764651009
Invite Link: https://discord.com/oauth2/authorize?client_id=634866217764651009&permissions=2080894207&scope=bot+applications.commands
Default Prefix & Giveaway Start Command: n!
